docs/usage/self-hosted-experimental.md

```diff
@@ -71,14 +71,18 @@ You can set the config file Renovate should read with the `RENOVATE_CONFIG_FILE`
 
 The process that runs Renovate must have the correct permissions to delete the config file.
 
+## `RENOVATE_X_DOCKER_HUB_TAGS_DISABLE`
+
+If set to any value, Renovate will stop using the Docker Hub API (`https://hub.docker.com`) to fetch tags and instead use the normal Docker API for images pulled from `https://index.docker.io`.
```

Comment:
```suggestion
By default, Renovate uses the _Docker Hub_ API (`https://hub.docker.com`) to fetch tags.

But when you set `RENOVATE_X_DOCKER_HUB_TAGS_DISABLE` to any value, Renovate will instead fetch tags from the normal _Docker_ API, for images pulled from `https://index.docker.io`.
```

The current text is hard to understand. I had to read multiple times to get the key point about the Docker API and Docker Hub API.

I'm trying to rewrite this so it's easier to understand:

- the default situation
- what changes when you use this config option
- split the sentence into multiple parts
- _emphasize_ the _Docker Hub_ and _Docker_ parts of the relevant sentence, as that is a key difference

Please double check my rewrite before applying it. :wink: